app/controllers/api/v1/application_controller.rb

```diff
@@ -48,14 +48,15 @@ def check_token_and_set_application
       private
 
       def authenticate
-        authenticate_or_request_with_http_token do |token, _options|
+        api_user = authenticate_or_request_with_http_token do |token, _options|
           ApiUser.find_by(token: token)
         end
+
+        Current.api_user = api_user
       end
 
       def current_api_user
```

Comment:
This method isn't called in all actions, meaning that `Current.api_user` doesn't get set, and therefore the `api_user` isn't set for audits.